internal/compiler-bridge/src/main/scala/xsbt/ClassName.scala

```diff
@@ -13,27 +13,19 @@ package xsbt
 
 import scala.tools.nsc.Global
 
-/**
- * Utility methods for creating (source|binary) class names for a Symbol.
- */
+/** Utility methods for creating (source|binary) class names for a Symbol. */
```

Comment:
Making short scaladoc one-line seems like a good idea. Not sure if it's worth doing massive code change for it, but this one I can get behind as more human-line.

Reply:
Funnily enough in light of my previous comment, this is a change I don't like because one-line scaladoc implicitly discourages documenting parameters. Even if your method initially doesn't have a parameter, if you add one then you have to completely change the scaladoc formatting from one-line to multiline.

Reply:
I'm fine with that 1-off cost being paid whenever a method adds its first parameter rather than the continuous cost of 2 extra pointless (IMO) lines per method (opening and closing the scaladoc).